admin/sql/CreateFunctions.sql

```diff
@@ -1066,7 +1079,13 @@ BEGIN
             SELECT release, date_year, date_month, date_day FROM release_unknown_country
         ) all_dates
         WHERE ' || condition ||
-        ' ORDER BY release, year NULLS LAST, month NULLS LAST, day NULLS LAST';
+        ' AND NOT EXISTS (
+          SELECT TRUE
+            FROM release
+           WHERE release.id = all_dates.release
+             AND status = 6
+        )
+        ORDER BY release, year NULLS LAST, month NULLS LAST, day NULLS LAST';
```

Comment:
The added subquery doesn't significantly affect the performance of the query. Planning and execution time are still both sub-1ms on floyd. (Tested with both cancelled and non-cancelled release IDs.)